sdk/azcore/policy_telemetry_test.go

```diff
@@ -37,7 +37,9 @@ func TestPolicyTelemetryWithCustomInfo(t *testing.T) {
 	defer close()
 	srv.SetResponse()
 	const testValue = "azcore_test"
-	pl := NewPipeline(srv, NewTelemetryPolicy(TelemetryOptions{Value: testValue}))
+	o := DefaultTelemetryOptions()
+	o.Value = testValue
```

Comment:
Are we checking that this is properly assigned in the test?

Reply:
Well we cant really since it's part of telemetry..

Reply:
We do check it below, it's just part of the `User-Agent` request header.

Reply:
Ah yes, I saw it. Thanks for pointing it out.